.evergreen/.evg.yml

```diff
@@ -2023,6 +2066,12 @@ buildvariants:
   tasks:
      - name: "perf"
 
+- name: rhel8-test-atlas
+  display_name: Atlas Cluster Tests
+  run_on: rhel80-large
+  tasks:
+    - test_atlas_task_group
```

Comment:
@rozza Should this run when a commit is merged into master, and if so, where could I set that up? (I did not see anything definite in the project settings.)

Reply:
Great question!

> Evergreen monitors a set of github repositories, waiting for new commits. When a new commit comes in or enough time has passed, Evergreen schedules builds for different variants (different OSes, compile flags, etc). <sup>[\[1\]](https://docs.devprod.prod.corp.mongodb.com/evergreen/Home/#how-it-works)</sup> 

So as the evergreen project monitors the master branch, all build variants will run by default. 

See [limiting when a task will run](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#limiting-when-a-task-will-run) to learn about opting out. One such example is the publish release task which opts out by setting `git_tag_only: true`.

Our evg.yml dates back from early evergreen days - I find it hard to grok and I wonder if there are newer features that we don't use. (a thought for another day).